batch7crud-roll4/src/main/java/com/lftechnology/batch7crud/dao/EmployeeDao.java

```diff
@@ -0,0 +1,136 @@
+package com.lftechnology.batch7crud.dao;
+
+import static com.lftechnology.batch7crud.constant.QueryConstants.*;
+import static com.lftechnology.batch7crud.constant.EntityConstants.*;
+
+import com.lftechnology.batch7crud.exception.DataException;
+import com.lftechnology.batch7crud.model.Employee;
+import com.lftechnology.batch7crud.util.DbConnection;
+
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * Created by pratishshr on 1/14/16.
+ */
+public class EmployeeDao {
```

Comment:
The methods in DAO need to communicate when something does not happen as expected. What if no records are deleted or if more than 1 record is updated, or no record is created?